01-js-intro-and-jquery/index.js

```diff
@@ -0,0 +1,46 @@
+$(document).ready(function (){
+	$('#sec').hide();
+	$('#sec').fadeIn(2000);
+	$('.alias').focus();
+	$('button').click(function (){
+		var nombre = $(".alias").val();
+		$.get("http://bootcamp.aws.af.cm/welcome/" + nombre, respond).error(ERROR);
```

Comment:
I don't like the callback function name "ERROR", although it doesn't seem to be part of the reserved keyword list, it looks like a global variable. Try to use more descriptive name like for example "onSuccess".